### DISCUSSION TITLE
信息栏增加功能

### DISCUSSION THREAD

**Discussion by ljs-fjmu-edu-cn** [MuiseDestiny/zotero-style#507] [Ideas]
![style](https://github.com/MuiseDestiny/zotero-style/assets/140003842/bd6170e0-59e9-40d8-a5ae-29e36364fdcd)
信息栏，右键增加功能：
1、作者比较多个时可折叠
2、摘要较长时可折叠
3、一些不用的字段可自主决定显示不显示，比如系列相关的3个，档案相关的5个

**Comment by MuiseDestiny** [comment]
这个是官方应该做的事情。

**Comment by ljs-fjmu-edu-cn** [comment]
> 这个是官方应该做的事情。

期待
